florist/api/monitoring/metrics.py

```diff
@@ -119,3 +122,40 @@ def wait_for_metric(
         retry += 1
 
     raise Exception(f"Metric '{metric}' not been found after {max_retries} retries.")
+
+
+def get_subscriber(channel: str, redis_host: str, redis_port: str) -> PubSub:
+    """
+    Return a PubSub instance with a subscription to the given channel.
+
+    :param channel: (str) The name of the channel to add a subscriber to.
+    :param redis_host: (str) the hostname of the redis instance.
+    :param redis_port: (str) the port of the redis instance.
+    :return: (redis.client.PubSub) The PubSub instance subscribed to the given channel.
+    """
+    redis_connection = redis.Redis(host=redis_host, port=redis_port)
+    pubsub: PubSub = redis_connection.pubsub()  # type: ignore[no-untyped-call]
```

Comment:
Is the ignore just because redis doesn't have type hints? Or why do we need these

Reply:
Yeah, Redis does not have it so mypy complains on our end.